pkg/smbplugin/Dockerfile

```diff
@@ -12,8 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM mcr.microsoft.com/aks/fundamental/base-ubuntu:v0.0.5
-RUN apt-get update && apt-get install -y ca-certificates cifs-utils util-linux e2fsprogs mount udev xfsprogs
+FROM golang:1.13.10-alpine3.10 as builder
+WORKDIR /go/src/sigs.k8s.io/csi-driver-smb
+ADD . .
+ARG LDFLAGS
+RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags "${LDFLAGS:--X sigs.k8s.io/csi-driver-smb/pkg/smb.driverVersion=latest}" -o _output/smbplugin ./pkg/smbplugin
+
+FROM us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:v2.1.0
+COPY --from=builder /go/src/sigs.k8s.io/csi-driver-smb/_output/smbplugin /smbplugin
+RUN clean-install util-linux e2fsprogs mount ca-certificates udev xfsprogs
```

Comment:
`ca-certificates cifs-utils util-linux e2fsprogs mount udev xfsprogs`

Reply:
Thank you for the review! I have updated the PR :)